best_practices/controllers.rst

```diff
@@ -164,13 +177,14 @@ manually. In our application, we have this situation in ``CommentController``:
 
 .. code-block:: php
 
+    use Doctrine\ORM\EntityManagerInterface;
+
     /**
      * @Route("/comment/{postSlug}/new", name = "comment_new")
      */
-    public function newAction(Request $request, $postSlug)
+    public function newAction(Request $request, $postSlug, EntityManagerInterface $em)
```

Comment:
> So, wdyt? It might be safer to change back to preferring $this->getDoctrine() throughout the docs?

I find `$this->getDoctrine()` easier to remember than a FQCN (which is not intuitive for someone who doesn't know doctrine internals) so I would revert, but of course that's subjective.

> What about suggest to extend from AbstractController with common features needed in controllers and use $this->getDoctrine() & Co.?

This would be great but I think this is too early, most tutorials use `$this->get()` and people are used to it. I'd wait for 3.3 features to be more commonly used.